web/src/app/shared/components/public/public.component.ts

```diff
@@ -25,20 +25,17 @@ export class PublicProjectsComponent implements OnInit {
     }
 
     ngOnInit(): void {
-        this.spinnerService.setProgressBar(true);
```

Comment:
Will it now be triggered by the `service`?

Reply:
Ignore me, that was part of another PR 🤦‍♂ . Here we are just moving the location of it.